src/node.rs

```diff
@@ -161,6 +195,9 @@ impl Sprawl {
         Ok(self.ids_to_nodes[&prev_id])
     }
 
+    /// Replaces the "n-th" child from the parent `node` with the new `child` node
```

Comment:
Same suggestion as above:
```suggestion
    /// Replaces the child at the given `index` from the parent `node` with the new `child` node
```